corehq/ex-submodules/couchexport/tests/test_excel_format_value.py

```diff
@@ -0,0 +1,305 @@
+import datetime
+
+from dateutil.tz import tzoffset
+from openpyxl.styles import numbers
+
+from django.test import SimpleTestCase
+
+from corehq.apps.export.const import MISSING_VALUE, EMPTY_VALUE
+from couchexport.util import get_excel_format_value
+
+
+class TestExcelFormatValue(SimpleTestCase):
+
+    def test_integers(self):
+        excel_format, value = get_excel_format_value('3423')
+        self.assertEqual(excel_format, numbers.FORMAT_NUMBER)
+        self.assertEqual(value, 3423)
+        self.assertEqual(type(value), int)
+
+        excel_format, value = get_excel_format_value('-234')
+        self.assertEqual(excel_format, numbers.FORMAT_NUMBER)
+        self.assertEqual(value, -234)
+        self.assertEqual(type(value), int)
+
+        excel_format, value = get_excel_format_value(324)
+        self.assertEqual(excel_format, numbers.FORMAT_NUMBER)
+        self.assertEqual(value, 324)
+        self.assertEqual(type(value), int)
```

Comment:
:heart: Excellent!

nit: these could be DRY'd up quite a bit. One suggestion is to use nose generator test:

```py
from testil import eq

def check(input, output, format, output_type):
    excel_format, value = get_excel_format_value(input)
    eq(excel_format, format)
    eq(value, output)
    eq(type(value), output_type)

def test_integers():
    yield check, '3423', 3423, numbers.FORMAT_NUMBER, int
    yield check, '-234', -234, numbers.FORMAT_NUMBER, int
    yield check, 324, 324, numbers.FORMAT_NUMBER, int

def test_decimal():
    yield check, '4.0345', 4.0345, numbers.FORMAT_NUMBER_00, float
    yield check, '-3.234', -3.234, numbers.FORMAT_NUMBER_00, float
    yield check, '5.032', 5.032, numbers.FORMAT_NUMBER_00, float

...
```